loom/script/native/lsNativeInterface.cpp

```diff
@@ -204,7 +204,8 @@ void NativeInterface::resolveScriptType(Type *type)
         }
     }
 
-    lmAssert(0, "Unable to resolve script type %s", type->getFullName().c_str());
+    type->setMissing("resolveScriptType");
```

Comment:
The base method sets `missing` to true, but I think I'll just make subclasses override if they want to add messages etc.